Services/CO.CDP.DataSharing.WebApi/DataService/DataService.cs

```diff
@@ -10,21 +10,90 @@ namespace CO.CDP.DataSharing.WebApi.DataService;
 
 public class DataService(IShareCodeRepository shareCodeRepository, IConnectedEntityRepository connectedEntityRepository) : IDataService
 {
+    private const string ExclusionsSectionName = "Exclusions";
+    private const string FinancialInformationSectionName = "Financial Information";
+    private const string QualificationsSectionName = "Qualifications";
+    private const string TradeAssurancesSectionName = "Trade assurances";
+
     public async Task<SharedSupplierInformation> GetSharedSupplierInformationAsync(string shareCode)
     {
         var sharedConsent = await shareCodeRepository.GetByShareCode(shareCode)
                             ?? throw new ShareCodeNotFoundException(Constants.ShareCodeNotFoundExceptionMessage);
 
+        var exclusions = sharedConsent.AnswerSets.Where(a => a.Section.Title == ExclusionsSectionName).ToArray();
```

Comment:
Shouldn't we generalize forms implementation in pdf?
We can excluded ShareCode using flag Declaration type

Reply:
Not sure what you mean. I have generalized it such that all form sections are now output in the PDF except declarations.

Reply:
Cool what you did is what I meant :)